bigchaindb/common/transaction.py

```diff
@@ -155,26 +155,26 @@ def _fulfillment_to_details(fulfillment):
     raise UnsupportedTypeError(fulfillment.type_name)
 
 
-def _fulfillment_from_details(data):
+def _fulfillment_from_details(data, _depth=0):
```

Comment:
There isn't an instance where converting a fulfillment to details would crash the system, since all transactions enter in a serialized form and are deserialised. It's only the other way around when building transactions, ie in the driver, and I'm not very worried about the use case of making a 100 deep threshold condition.